app/controllers/users/registrations_controller.rb

```diff
@@ -1,9 +1,12 @@
 module Users
   class RegistrationsController < Devise::RegistrationsController
     def create
```

Comment:
That's more readable, but kinda the same :)
I guess we can leave it as it is for now and refactor in the future once the need arises